config/environments/development.rb

```diff
@@ -35,7 +35,8 @@
   # Checks for improperly declared sprockets dependencies.
   # Raises helpful error messages.
   config.assets.raise_runtime_errors = true
-
+	
+  #config.force_ssl = true
```

Comment:
Don't force ssl in development cause we want to be able to test with http on localhost. When we move to production we already have set config.force_ssl = true in production.rb.